Rubberduck.Core/UI/Command/MenuItems/CommandBars/AppCommandBarBase.cs

```diff
@@ -184,7 +202,6 @@ public void RemoveCommandBar()
                     Item.Delete();
                     Item.Dispose();
```

Comment:
The dispose call is not necessary here any more - the assignment to null handles that now (as intended).